python/cudf/cudf/tests/test_csv.py

```diff
@@ -128,6 +128,46 @@ def make_all_numeric_dataframe():
     )
 
 
+def make_all_numeric_extremes_dataframe():
+    # integers 0,+1,-1,min,max
+    # float 0.0, -0.0,+1,-1,min,max, nan, esp, espneg, tiny, [-ve values]
+    df, gdf_dtypes, pdf_dtypes = make_all_numeric_dataframe()
+    df = pd.DataFrame()
+
+    for gdf_dtype in gdf_dtypes:
+        np_type = pdf_dtypes[gdf_dtype]
+        if np.issubdtype(np_type, np.integer):
+            itype = np.iinfo(np_type)
+            extremes = [0, +1, -1, itype.min, itype.max]
+            df[gdf_dtype] = np.array(extremes * 4, dtype=np_type)[:20]
+        else:
+            ftype = np.finfo(np_type)
+            extremes = [
+                0.0,
+                -0.0,
+                +1,
+                -1,
+                np.nan,
+                -np.nan,
+                # ftype.min, # can't enable because of truncation issue #6235
```

Comment:
Maybe also just mention in this comment a `TODO` to uncomment as `#6235` is fixed?

Reply:
done.